data/01-07-20.json

```diff
@@ -0,0 +1,166 @@
+[
+    {
+        "username": " jinritv",
+        "displayName": "진리",
+        "steamerName": "진리 (jinritv)",
+        "partner": true,
+        "name": "jinri lee",
+        "dateStarted": null,
+        "avgViewerCountPerStream": 60,
+        "followers": 3615,
+        "fulltime": false,
+        "avgStreamLength": null,
+        "age": 21,
+        "DOB": 1999,
+        "languages": [
+            "KR",
+            "EN"
+        ],
+        "ethnicities": [
+            "Korean",
+            "American"
+        ],
+        "location": [
+            "Wisconsin"
+        ],
+        "content": [
+            "raspberry pi research",
+            "stream forcasting",
+            "research other streamers data capture setup",
+            "just chatting"
+        ],
+        "mainGame": "Just Chatting (studying?)",
+        "collaborations": null,
+        "voice": [
+            "Low"
+        ],
+        "cam": true,
+        "frequency": false,
```

Comment:
hope "frequency":  3 times/week
could adjust it

Reply:
Fixed this! before frequency was parsing to a boolean which doesn't make sense, now it should be a string.
(this fix change wont be reflected in this PR)